packages/react-dev-utils/README.md

```diff
@@ -166,6 +166,43 @@ compiler.plugin('done', function(stats) {
 });
 ```
 
+#### `formatter(results: Object): string`
+
```

Comment:
Maybe "This is our custom ESLint formatter that integrates well with Create React App console output. You can remove it and use the default one instead."